### PR TITLE
eACL tables for `$Object:` filter

### PR DESCRIPTION
**Pull request by EliChin**
Add two tables for setting eACL (as a response to issue [#949](https://github.com/nspcc-dev/neofs-node/issues/949)):
- combination of NeoFS verbs and object attributes
- spawning requests

Minor language check

#62 
Signed-off-by: Elizaveta Chichindaeva <elizaveta@nspcc.ru>

### CODE REVIEW

File: 01-arch/07-acl.md [hunk 1]
```diff
@@ -103,6 +103,34 @@ and can be presented in different intermediate formats, like JSON, for the users
...
+|version        |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|objectID       |    v    |    v    |    v    |    v    |    -    |    v    |    v    |
+|containerID    |    v    |    v    |    v    |    v    |    v    |    v    |    v    |
+|ownerID        |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|creationEpoch  |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|payloadLength  |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|payloadHash    |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|objectType     |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|homomorphicHash|    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|// User headers|    v    |    v    |    v    |    -    |    -    |    -    |    -    |
```
Comment (realloc) [inline 01-arch/07-acl.md:119]: Why is '//' here?

File: 01-arch/07-acl.md [hunk 2]
```diff
@@ -103,6 +103,34 @@ and can be presented in different intermediate formats, like JSON, for the users
 }
 ```
 
+Note that some filters with `$Object` prefix are not suitable for making denying rules on certain operations. There may be an undefined behaviour on some combinations of NeoFS verbs and object attributes when eACL is set.
+
+|$Object:       |   GET   |  HEAD   |   PUT   | DELETE  | SEARCH  |  RANGE  |RANGEHASH|
+|---------------|---------|---------|---------|---------|---------|---------|---------|
+|version        |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
```
Comment (realloc) [inline 01-arch/07-acl.md:110]: Need to make a clarification on what does `v` and `-` mean here.

File: 01-arch/07-acl.md (hunk 1 above)
Comment (EliChin) [reply 01-arch/07-acl.md:119]: That's how it was written in [the issue](https://github.com/nspcc-dev/neofs-node/issues/949#issuecomment-956104749).
I think @alexvanin can respond to this comment.

File: 01-arch/07-acl.md (hunk 2 above)
Comment (EliChin) [reply 01-arch/07-acl.md:110]: We can decide on better marks.
What about using `OK` for `v` and `UB` (as in Undefined Behavior) for `-`?

**Review by realloc — APPROVED** [review]

**Pull request merged**